c_glib/arrow-glib/type.h

```diff
@@ -116,7 +118,10 @@ typedef enum {
   GARROW_TYPE_LARGE_LIST,
   GARROW_TYPE_MONTH_DAY_NANO_INTERVAL,
   GARROW_TYPE_RUN_END_ENCODED,
-  /* TODO: Remove = 43 when we add STRING_VIEW(39)..LARGE_LIST_VIEW(42). */
+  GARROW_TYPE_STRING_VIEW,
```

Comment:
Could you add this when we add support string view data type?

```suggestion
```

Reply:
Thanks Removed.